.env

```
@@ -0,0 +1,2 @@
+NODEMAILER_EMAIL=diket68477@bitfami.com
+NODEMAILER_PW=12345678
```

dotenv-linter (3.3.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)